content/sensu-core/2.0/reference/silencing.md

```diff
@@ -36,63 +36,44 @@ description  | The name of the check the entry should match
 required     | true, unless `subscription` is provided
 type         | String
 example      | {{< highlight shell >}}"check": "haproxy_status"{{</ highlight >}}
-example      | {{< highlight shell >}}"check": "haproxy_status", "subscription" : "load_balancer"{{</ highlight >}}
+
 
 subscription | 
 -------------|------ 
 description  | The name of the subscription the entry should match 
 required     | true, unless `check` is provided
 type         | String
 example      | {{< highlight shell >}}"subscription": "entity:i-424242"{{</highlight>}}
-example      | {{< highlight shell >}}
-  "subscription": "entity:i-424242",
-  "check" : "haproxy_status"
-{{</ highlight >}}
```

Comment:
I removed multiple examples and examples with multiple attributes in order to be more consistent with 1.x docs and 
slim this document. Let me know if you don't think it's a good idea!